report/web.py

```diff
@@ -510,18 +511,41 @@ def _write_benchmark_sample(self, benchmark_id: str, sample_id: str):
       print(f'Failed to write sample/{benchmark_id}/{sample_id}:\n{e}')
 
 
+def _parse_arguments() -> argparse.Namespace:
+  """Parses command line args."""
+  parser = argparse.ArgumentParser(
+      description='Parse arguments to launch the web report generation.')
```

Comment:
This should be a description of what the program does. Maybe something like "Report generation tool reads raw experiment output files and generates a report in the form of HTML files in a directory hierarchy."

Reply:
Sounds good, done